core/src/test/java/io/grpc/internal/CensusModulesTest.java

```diff
@@ -1148,4 +1166,92 @@ private static void assertNoClientContent(StatsTestUtils.MetricsRecord record) {
     assertNull(record.getMetric(DeprecatedCensusConstants.RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES));
     assertNull(record.getMetric(DeprecatedCensusConstants.RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES));
   }
+
+  @Deprecated
+  @Test
+  public void newTagsPopulateOldViews() throws InterruptedException {
+    StatsComponent localStats = new StatsComponentImpl();
```

Comment:
Why do you need this? As an alternative consider:
```java
private static final ViewManager viewManager = Stats.getViewManager();
...
viewManager.registerView(...);
```

Reply:
I didn’t want the unit test to modify global state.

Reply:
OK that makes sense. In this case you don't need to modify the dependencies in build.